File: CLASS 2/11050 이항 계수 1.c
#define _CRT_SECURE_NO_WARNINGS // scanf사용시 에러무시
#include <stdio.h>

int main(void)
{
    int N = 0, K = 0;
    scanf("%d %d", &N, &K);

    int numerator = 1;

    for (int i = 0; i < K; i++)
    {
        numerator *= N;
        N--;
    }

    int denominator = 1;

    while (K > 0)
    {
        denominator *= K;
        K--;
    }

    printf("%d", numerator / denominator);
    return (0);
}

File: CLASS 2/11050 이항 계수 1.cs
using System;

namespace ConsoleApp2
{
    public class Program
    {
        static void Main(string[] args)
        {
            string str = Console.ReadLine();
            string[] str_split = str.Split(' ');

            int N = int.Parse(str_split[0]);
            int K = int.Parse(str_split[1]);

            int numerator = 1;
            for (int i = 0; i < K; i++)
            {
                numerator *= N;
                N--;
            }

            int denominator = 1;
            while (K > 0)
            {
                denominator *= K;
                K--;
            }

            Console.WriteLine(numerator/denominator);
        }
    }
}
